nxc/protocols/ssh.py

```diff
@@ -32,28 +44,23 @@ def proto_logger(self):
                 "hostname": self.hostname,
             }
         )
-        logging.getLogger("paramiko").setLevel(logging.WARNING)
 
     def print_host_info(self):
-        self.logger.display(self.remote_version)
+        self.logger.display(self.remote_version if self.remote_version else "Unknown SSH version, skipping...")
```

Comment:
I think we should just set the version to "Unknown", since the variable is remote version, we already know what it is.

Reply:
Like this?

Reply:
Hmm, sorry for my comprehension